packages/block-library/src/categories/block.json

```diff
@@ -39,6 +39,14 @@
 		"showLabel": {
 			"type": "boolean",
 			"default": true
+		},
+		"order": {
+			"type": "string",
+			"default": "asc"
+		},
+		"orderBy": {
+			"type": "string",
+			"default": "name"
```

Comment:
I kept the default query to prevent unexpected results.

Reply:
I was also wondering if we should create a `query` attribute like the query loop has and include properties there like `taxonomy`. But I am not sure how it would work with backward compatibility.